docs/pages/blog/2023-chamonix-retreat.md

```diff
@@ -0,0 +1,90 @@
+---
+title: MUI's team retreat in Chamonix
```

Comment:
@samuelsycamore's [post about the previous retreat](https://mui.com/blog/2022-tenerife-retreat/) had a slightly different title ("MUI's company retreat in Tenerife: a recap"). Should we standardize it, meaning this one would turn to "MUI's company retreat in Chamonix: a recap"? No strong preference for any of the two but probably continuing the trend would be good 👌

Reply:
Thanks for all your support, @danilo-leal! Happy to standardise the title. :)